Common/POpen.cpp

```diff
@@ -100,6 +100,10 @@ POpen::POpen(const std::string& cmd) : m_Command(cmd)
 				//WaitForSingleObject(m_ProcInfo.hProcess, msWait);
 				//GetExitCodeProcess(m_ProcInfo.hProcess, &exitCode);
 			}
+			else if (exitCode == 0)
+			{
+				break;
+			}
 			else
```

Comment:
I think it could be even more readable to remove the
`else if (exitCode == 0)`
above and just use
`else if (exitCode != 0)`
to throw the exception

Reply:
In fact, I still don't understand the whole commit; 
`case WAIT_FAILED:`
which, I guess, should never happen, or never return exit code 0

Reply:
Clean-up done